### PR TITLE
update analytics

### PR DESCRIPTION
**Pull request by youngfreezyVA**
https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/23158

### CODE REVIEW

**Review by LinnJS — APPROVED** [review]
Pulled on a local branch and works as expected

File: src/applications/caregivers/containers/App.jsx [hunk 1]
```diff
@@ -15,23 +15,26 @@ const App = ({ loading, location, children }) => {
...
+              label,
+              e.target.value,
+              'LABEL AND VALUE SENT TO RECORD EVENT',
+            );
             recordEvent({
-              'caregivers-check-box-label': label,
-              'caregivers-check-box-clicked': e.target,
-              'caregivers-check-box-value-selected': e.target.value,
+              'caregivers-radio-label': label,
+              'caregivers-radio-clicked': e.target,
```
Comment (LinnJS) [inline src/applications/caregivers/containers/App.jsx:30]: 🔥

**Review by timwright12 — APPROVED** [review]

File: src/applications/caregivers/containers/App.jsx [hunk 2]
```diff
@@ -15,23 +15,26 @@ const App = ({ loading, location, children }) => {
...
+        const radios = document.querySelectorAll('input[type="radio"]');
+        for (const radio of radios) {
+          radio.onclick = e => {
+            const label = e.target.nextElementSibling.innerText;
+            // eslint-disable-next-line no-console
+            console.log(
+              label,
+              e.target.value,
+              'LABEL AND VALUE SENT TO RECORD EVENT',
+            );
```
Comment (ncksllvn) [inline src/applications/caregivers/containers/App.jsx:27]: Can probably remove this `console.log`

File: src/applications/caregivers/containers/App.jsx [hunk 3]
```diff
@@ -15,23 +15,26 @@ const App = ({ loading, location, children }) => {
...
     () => {
       if (!loading) {
-        const checkBoxes = [
-          ...document.querySelectorAll('[value="Y"]'),
-          ...document.querySelectorAll('[value="N"]'),
-        ];
-        for (const checkbox of checkBoxes) {
-          checkbox.onclick = e => {
-            const label = e.target.closest('fieldset')?.firstChild?.innerText;
+        const radios = document.querySelectorAll('input[type="radio"]');
```
Comment (ncksllvn) [inline src/applications/caregivers/containers/App.jsx:18]: I don't understand the problem here well enough but it seems a little risky to me to bind onclick handlers to all radio boxes on the page

**Review by ncksllvn — COMMENTED** [review]
Wondering if there is an issue we can link to, or if we can populate the PR description to help give some context? Are you just lending a hand to the Caregivers team?

**Comment by youngfreezyVA** [comment]
> Wondering if there is an issue we can link to, or if we can populate the PR description to help give some context? Are you just lending a hand to the Caregivers team?

yes just for this sprint.

**Comment by LinnJS** [comment]
@ncksllvn here is the original ticket this PR is meant to patch

https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/23158

**Pull request merged**